wallet/db.c

```diff
@@ -177,6 +177,14 @@ char *dbmigrations[] = {
     "ALTER TABLE payments ADD COLUMN payment_preimage BLOB;",
     /* We need to keep the shared secrets to decode error returns. */
     "ALTER TABLE payments ADD COLUMN path_secrets BLOB;",
+    /* Create time-of-payment of invoice, default already-paid
+     * invoices to current time. */
+    "ALTER TABLE invoices ADD paid_timestamp INTEGER;",
+    "CREATE INDEX invoices_paid_timestamp"
+    "    ON invoices(paid_timestamp);",
+    "UPDATE invoices"
```

Comment:
Why create this index?

Reply:
I was thinking it would be useful for sorting according to timestamp, but I realize now that sorting according to `pay_index` would be the same anyway.  I will remove this.